.github/workflows/release.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - ubuntu-20.04
+          - ubuntu-22.04
```

Comment:
Other jobs are already using `ubuntu-latest`, so I believe we can safely use it here as well for consistency. 
What do you think?